### DISCUSSION TITLE
need some help on charles

### DISCUSSION THREAD

**Discussion by D-06** [Lost-MSth/Arcaea-server#67] [Q&A] [answered]
i tried modify arcaea to make them accept charles certificate but somehow it cant detect anything from arceea(like server domain) and its just ignore my request too

**Comment by Lost-MSth** [comment, marked as answer]
hmmm... Maybe you used WiFi proxy? It doesn't work.
A vpn app is needed.

**Reply by D-06** [reply]
its work but its cant connect to my server this time

**Reply by Lost-MSth** [reply]
Emmmm..  it works for the client under 3.6.0 version.

Additionally, the client may try some strange ways to connect when the first attempt fails. So restart the client and try again.

**Reply by D-06** [reply]
so there is no way to connect to newer version?

**Reply by Lost-MSth** [reply]
You said you do something to make the cert accepted. So why no ways?
Kill the ssl pinning and the ssl cert verification. Then use a vpn app to connect the client to Charles. That's all.

**Reply by D-06** [reply]
thats is the difficult part since i cant find ssl pinning or verification function

**Reply by Lost-MSth** [reply]
Well, it is really difficult to find them.

However, I heard of another way. Just search the url 'arcapi....' and change it arbitrarily. Maybe work maybe not. I haven't tried.

**Reply by D-06** [reply]
huh it doesnt work either

**Reply by D-06** [reply]
ALLOW_ALL_HOSTNAME_VERIFIER
maybe this?

**Reply by Lost-MSth** [reply]
Actually symbols were deleted in the version above 3.10.0. You may need a low version for reference.

**Reply by D-06** [reply]
I think im gonna stick with 3.5.5 since this take longer to make than i thought

**Reply by D-06** [reply]
But thanks anyway
